src/mixins/message-mixin.ts

```diff
@@ -75,6 +75,7 @@ const messageMixin = <MinxinBase extends typeof PuppetSkeleton & CacheMixin>(bas
     abstract messageSendMiniProgram (conversationId: string, miniProgramPayload: MiniProgramPayload) : Promise<void | string>
     abstract messageSendPost        (conversationId: string, postPayload: PostPayload)               : Promise<void | string>
     abstract messageSendText        (conversationId: string, text: string, mentionIdList?: string[]) : Promise<void | string>
+    abstract messageSendTextV2      (conversationId: string, text: string | string[], mentionIdList?: string[]) : Promise<void | string>
```

Comment:
I think we can just upgrade the `messageSendText` instead of adding a new `V2`, here are the reasons:

1. the old Wechaty will not use the old Puppet
2. the new Wechaty will use the new Puppet
3. the new Wechaty will never use the old Puppet

So it seems that there's no chance that a Wechaty user will use `string[]` to call the old `messageSendText` function?

Reply:
OK. I'll create a new PR when wechaty 2.0 branch is created.

Reply:
May I know whether you agree with my comment or not?

Thanks!

Reply:
Totally. Unlike grpc and service, wechaty and puppet will never be mismatched.

Reply:
Good to know! Cheers!